src/osd/ECUtil.h

```diff
@@ -32,6 +32,8 @@ const uint64_t CHUNK_INFO = 8;
 const uint64_t CHUNK_PADDING = 8;
 const uint64_t CHUNK_OVERHEAD = 16; // INFO + PADDING
 
+const std::string HINFO_KEY = "hinfo_key";
```

Comment:
@tchaikov Good question.

I just wanted to keep *.cc file as much as possible. Since that, I moved static globally declared param in *.cc to header file.